doc-content/enterprise-only/processes/pizza-end-node-proc.adoc

```diff
@@ -8,10 +8,4 @@ Add an end node to signify the end of the pizza order process.
 .Outgoing connection from the Confirmation user task
 image::processes/create-end-order.png[Creating an outgoing connection from the Confirmation user task]
 
-. Convert the new task to a user task.
```

Comment:
Not sure why is it removed.

Reply:
@hasys, Here we are creating an End Event, not a Task.